server/src/serializers/UserSerializer.js

```diff
@@ -0,0 +1,12 @@
+class UserSerializer {
+    static serializeUser(user) {
+        const cleanedUserObject = {}
+        const allowedAttributes = ["id", "email", "createdAt"]
+        for(const attribute of allowedAttributes){
```

Comment:
fix the whitespace of line 5
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--before-blocks
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--around-keywords